### PR TITLE
Send information about action type in cookies

### PR DESCRIPTION
**Pull request by xapiksw**

### CODE REVIEW

**Comment by xapiksw** [comment]
This PR depends on https://github.com/scandipwa/scandipwa-base/pull/38 and https://github.com/scandipwa/base-theme/pull/232

File: src/Controller/Pwa.php [hunk 1]
```diff
@@ -35,7 +40,21 @@ class Pwa extends Action implements HttpGetActionInterface, HttpPostActionInterf
...
+    /**
+     * @var CookieManagerInterface
+     */
+    private $cookieManager;
+
+    /**
+     * @var CookieMetadataFactory
+     */
+    private $cookieMetadataFactory;
+    /**
```
Comment (alfredsgenkins) [inline src/Controller/Pwa.php:53]: Make sure enters are correct.

**Review by alfredsgenkins — APPROVED** [review]

**Comment by alfredsgenkins** [comment]
Needs QA to be merged

**Comment by alfredsgenkins** [comment]
Closing because the idea was revoked.

**Pull request closed without merging**